libraries/helpers.rb

```diff
@@ -57,14 +57,16 @@ def shell_is_valid?(shell_path)
     # Validates passed id.
     #
     # @return [Numeric, String]
-    # handles checking whether uid was specified as a string
+    # handles basic input validation and checking 
+    # whether uid was specified as a string
     def validate_id(id)
+      return if id.nil? || id == 0
```

Comment:
Leave a newline after a guard. Also i'd rethink this method generally as there's a mixture of types going on and nothing to handle them.